dts/bindings/battery/intel,sbs_battery.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 Intel Corporation
```

Comment:
this should come with the driver, not part of this PR introducing an API

Reply:
Yes. This is a question i have too. If we have to keep a driver implementation out of tree, how do we express it in dts?

Reply:
You can also have DTS data out of tree.